BridgeSDK/BridgeAPI/SBBAuthManager.m

```diff
@@ -270,10 +273,26 @@ + (instancetype)authManagerWithBaseURL:(NSString *)baseURL
     return authManager;
 }
 
+- (void)clearOldKeychainOnReinstall
+{
+    NSUserDefaults *defaults = BridgeSDK.sharedUserDefaults;
```

Comment:
This is a little confusing. You mention Core data as the cause of the issue but you clear it using NSUserDefaults. Can you please update the comment to reflect that? Thanks!